values-nl/strings.xml

```diff
@@ -62,31 +62,31 @@
     <!--______GENERAL______-->
         <!--LABELS-->
             <string name="app_name">Open GApps</string>
-            <string name="label_cancel">Annuleren</string>
+            <string name="label_cancel">Annuleer</string>
             <string name="label_no">Nee</string>
             <string name="label_yes">Ja</string>
-            <string name="label_close">Sluiten</string>
+            <string name="label_close">Sluit</string>
             <string name="label_adblock_detected">Adblocker gedetecteerd</string>
             <string name="label_permission_denied">Toestemming geweigerd</string>
-            <string name="permission_granted">Toestemming verkregen! Dank!</string>
+            <string name="permission_granted">Toestemming verkregen, bedankt!</string>
             <string name="label_grant_permission">Toestemming geven</string>
             <string name="navigation_drawer_close">Navigatie sluiten</string>
             <string name="navigation_drawer_open">Navigatie openen</string>
-            <string name="label_accept">Accepteren</string>
+            <string name="label_accept">Accepteer</string>
             <string name="label_accepted">Geaccepteerd</string>
-            <string name="label_decline">Weigeren</string>
+            <string name="label_decline">Weiger</string>
             <string name="label_delete">Verwijder</string>
-            <string name="title_optional_app_update">App update beschikbaar</string>
+            <string name="title_optional_app_update">App-update beschikbaar</string>
             <string name="ms_prev">Vorige</string>
             <string name="ms_next">Volgende</string>
             <string name="ms_end">Klaar</string>
         <!--EXPLANATION-->
-            <string name="explanation_permission_denied">Om goed te kunnen functioneren dient u OpenGApps toestemming te geven uw bestandssysteem te gebruiken. Logisch, toch?</string>
-            <string name="explanation_adblock_detected">Adblocker gedetecteerd!? Advertenties kunnen vervelend zijn, maar houden dit project en de ontwikkelaars in leven! Wilt u a.u.b. de adblocker uitschakelen of anders een donatie schenken?</string>
+            <string name="explanation_permission_denied">Om goed te kunnen functioneren dient u Open GApps toestemming te geven uw bestandssysteem te gebruiken. Logisch, toch?</string>
+            <string name="explanation_adblock_detected">Adblocker gedetecteerd?! Advertenties kunnen vervelend zijn, maar houden dit project en de ontwikkelaars in leven! Wilt u a.u.b. de adblocker uitschakelen of anders een donatie schenken?</string>
```

Comment:
!? is populairder en wordt meer gebruikt dan ?!  :-/ het is om het even (maar prefeer iig consistentie tussen alle talen ;) )